concourse/scripts/set_security_groups_from_manifest.rb

```diff
@@ -47,17 +47,10 @@ def existing_security_groups
   end
 
   def fetch_security_groups
-    groups = []
+    response = `cf curl /v3/security_groups?per_page=1000`
+    abort "security_groups request failed" unless $CHILD_STATUS.success?
 
-    security_groups = `cf security-groups`
-    abort security_groups unless $CHILD_STATUS.success?
-
-    security_groups.each_line do |line|
-      if line =~ /\A#\d+\s+(\S+)\s*/
-        groups << Regexp.last_match(1)
-      end
-    end
-    groups
+    JSON.parse(response)["resources"].map {|sg| sg["name"]}
```

Comment:
I like this. It's simple, concise, and self-explanatory

Reply:
Too concise for rubocop it turns out